pkg/local/authorization.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/cmsgov/easi-app/pkg/models"
 )
 
-const testEUAID = "ABCD"
+const testEUAID = "GZVA"
```

Comment:
please switch this back to ABCD.

Reply:
Good catch. We should note, however, that this will mean we can't fetch this user by EUAID from LDAP (since that's not a real user)